src/Sulu/Component/Contact/Tests/Unit/SmartContent/ContactDataProviderTest.php

```diff
@@ -160,7 +158,7 @@ public function testNullSortBy(): void
         $this->assertEquals($dataItems, $result->getItems());
     }
 
-    public function resourceItemsDataProvider()
+    public static function resourceItemsDataProvider(): array
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes